extraMbedtls/generate-pkcs7.c

```diff
@@ -49,13 +49,14 @@ extern int verbose;
 typedef struct PKCS7Info {
 	unsigned char **crts; // signing crt DER
 	size_t *crtSizes;
-	unsigned char **keys; // signing key DER
+	unsigned char **keys; // signing key DER or signatures (depends on alreadySignedFlag)
```

Comment:
So, I'm not fully up on the pkcs7 implementation for mbedtls, but make sure if you are adding features to the pkcs7 generation code for new features in secvarctl, that those changes make sense for mbedtls as well. 

It may be suggested that rolling pre-signed data into a pkcs7 blob should be handled in glue code outside of the library (i.e. in secvarctl) instead of in the library itself. Not suggest you change this now, but more of a warning. Maybe look at other implementations of similar algos/specs/etc and see if there's precedence for this?